examples/esp32/xively_demo/main/main.c

```diff
@@ -221,22 +226,12 @@ int8_t app_wifi_station_init( user_data_t* credentials )
     printf( "\n|               Initializing WiFi as Station             |" );
     printf( "\n|********************************************************|\n" );
 
-#if USE_HARDCODED_CREDENTIALS
-    wifi_config_t wifi_config =
-    {
-        .sta =
-        {
-            .ssid = USER_CONFIG_WIFI_SSID, .password = USER_CONFIG_WIFI_PWD
-        }
-    };
-#else
```

Comment:
This was redundant -- Hardcoded credentials are already in the *credentials struct